File: multimodule-example/util/src/test/java/xxx/HelloWorldTest.java
package xxx;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class HelloWorldTest {
    @Test
    void shouldGreetWarmly() {
        final var greeter = new HelloWorld();

        assertEquals("Hello, world!", greeter.greet());
    }
}

File: kotlin/src/test/kotlin/hm/binkley/exemplar/kotlin/HelloWorldTest.kt
package hm.binkley.exemplar.kotlin

import hm.binkley.exemplar.kotlin.HelloWorld.Companion.GREETING
import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class HelloWorldTest {
    @Test
    fun shouldGreet() {
        assertEquals(
            GREETING, HelloWorld()
                .greet()
        )
    }
}
